### PR TITLE
New catalogue for NatMap using Terria v8

### PR DESCRIPTION
**Pull request by meh9**
I think it's ready to go.

### CODE REVIEW

File: natmap/root.js [hunk 1]
```diff
@@ -1,9 +1,54 @@
...
+          id: "6EKwTvLj",
+          type: "group",
+          members: Agriculture.members
+        },
+        {
+          name: "Communications",
+          id: "EjSkCmuD",
+          type: "group",
+          members: Communications.members
+        }
```
Comment (tephenavies) [inline natmap/root.js]: This could just be:
```suggestion
        Agriculture,
        Communications
```

Comment (tephenavies) [reply natmap/root.js]: I think

File: natmap/root.js [hunk 2]
```diff
@@ -1,9 +1,54 @@
...
+Communications.members = Communications.members
+                            .filter(m => m.name !== "ABC Photo Stories (2009-2014)")
+                            .filter(m => m.name !== "ABC Photo Stories by date");
 
+module.exports = {
+  catalog: [
+    {
+      name: "National Datasets",
+      type: "group",
+      members: [
```
Comment (tephenavies) [inline natmap/root.js]: In the same spirit as Agriculture and Communications you could `getFromCatalogPath` to fetch NationalDatasets, set: 
```
NationalDatasets.members = [Agriculture, Communications];
```

then:
```
return {
  catalog: [NationalDatasets]
};
```

File: natmap/root.js (hunk 1 above)
Comment (meh9) [reply natmap/root.js]: Aww yeah now we're cooking with gas! This is the sort of thing I was hoping for, thanks, will give it a shot in the morning.

**Pull request merged**